<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingswindow</class>
 <widget class="QWidget" name="settingswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="settingsTitle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>500</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="straightKeyToggle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Paddle Toggle</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="volumeTitle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Volume</string>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>180</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="wpmTitle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Words Per Minute:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="wpmSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QKeySequenceEdit" name="morseKeyBind">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSequenceLength">
    <longLong>1</longLong>
   </property>
  </widget>
  <widget class="QLabel" name="morseKeybindTitle">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>320</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Morse Code Input Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Default input mode is straight key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
